### ISSUE TITLE
ui: wallet locking has some issues

### ISSUE THREAD

**Issue by buck54321** [decred/dcrdex#2086]
I placed and settled a dcr-dextt order on a fresh harness. Fully executed. Went to the wallets view. Clicked lock.
![image](https://user-images.githubusercontent.com/6109680/215484430-c9c6854e-fc3d-4b7e-9d6c-29308a19d9ba.png)
Nothing happened. No error messages. No waiting spinner. Button didn't disappear. No console error.
Error messages logged to terminal.
```
[ERR] WEB: error locking dcr wallet: cannot lock dcr wallet with active swap negotiations
```
A couple of things need looking into.
1. Why can't I lock the wallet? Decred is not a `DynamicSwapOrRedemptionFeeChecker`. The order is fully executed.
2. How can we improve the UI/UX for this error?

**Comment by chappjc** [comment]
It's notable that one side was dextt and that it was a self-trade.  What was the status of the match on the side redeeming dextt?  Agree that shouldn't affect the match on the side redeeming dcr, but I wonder if the dextt *init* was still trying to get dynamic fees.

Ultimately, `db.MatchIsActive` is saying true for you for the *dcr wallet*, so for some reason I think that must indicate it hadn't changed the `core.UserMatch.Status` to `order.MatchConfirmed`.

**Comment by buck54321** [comment]
> but I wonder if the dextt init was still trying to get dynamic fees.

That could be. Part 1) might be resolved by #2088
